cirq-core/cirq/contrib/qasm_import/_parser.py

```diff
@@ -201,6 +201,12 @@ def __init__(self):
         'id': QasmGateStatement(
             qasm_gate='id', cirq_gate=ops.IdentityGate(1), num_params=0, num_args=1
         ),
+        'p': QasmGateStatement(
```

Comment:
AFAICT, this gate does not exist in the standard gate library of OpenQASM 2.0 (which is the version cirq supports). I checked the standard header in section 3.1 in [this paper](https://arxiv.org/abs/1707.03429). That said, the P gate is defined in the standard library of OpenQASM 3 (stdgates.inc), see section 4.1.1 in [this paper](https://arxiv.org/abs/2104.14722).

It's not wrong to include it for user's convenience, but I'm concerned this makes the world a little bit messier (instead of two relevant gatesets: OpenQASM 2.0 and OpenQASM 3.0 we'd now have three: OpenQASM 2.0, OpenQASM 3.0 and the chimera supported by cirq), so I'm leaning against it. What do others think? @MichaelBroughton @dabacon @dstrain115 @Strilanc 

---
(I used [this post](https://quantumcomputing.stackexchange.com/questions/22191/where-is-definitive-documentation-for-openqasm-2-0-and-3-0) for the source of truths about different versions of OpenQASM.)